examples/born/apbs-mol-fem-extmesh.in

```diff
@@ -3,13 +3,14 @@
 #############################################################################
 read 
 	mol pqr ion.pqr
-	mesh mcsf mesh.m
+	# WARN: The following no longer works
+	# mesh mcsf mesh.m
```

Comment:
If this no longer works, then does the example still have meaning?  Does it run to completion?

Before, the example was intended to show working FEM solver and working mesh loading.

Reply:
I put the `WARN` comment in to catch this. The mesh keyword is not documented in [READ](https://apbs.readthedocs.io/en/latest/using/input/read.html). I did not want to add something into the READ specification that is not documented or supported. Is `mesh` a file format that FEM uses?

Reply:
Yes, FEM used to support mesh input.  However, it's not a high-priority feature and I'm OK dropping it from the input file (if we also drop it from the code).

Reply:
> I put the `WARN` comment in to catch this. The mesh keyword is not documented in [READ](https://apbs.readthedocs.io/en/latest/using/input/read.html). I did not want to add something into the READ specification that is not documented or supported. Is `mesh` a file format that FEM uses?

Does the `WARN` comment generate any user-visible output?

Reply:
I created an issue to review the data dictionaries and update the examples.